graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/security/MultiReadRequestWrapper.java

```diff
@@ -0,0 +1,61 @@
+package org.jahia.modules.graphql.provider.dxm.security;
```

Comment:
Shouldn't we add these two classes to a separate module? @fpral says in the linked ticket

> we need to find a mitigation way for the following Jahia versions:
> 
>     Jahia 7.3.9 and 7.3.10
>     Jahia 8.0.3
>     Jahia 8.1.1 and 8.1.2
> 

Putting it into graphql-dxm-provider, we would need to create minor versions for all possible or used graphql-dxm-provider version on these mentioned Jahia versions. Having the filter in an own module we could more easily give it to customers, who found and are concerned by the known DoS vulnerability or even decide to pro-actively communicate that all customers should install it.

Reply:
Sure, I can try to move it out into a separate module